### PR TITLE
SUPESC-606 Updated Availability Notification IG with stores to Yves host mapping configuration.

### PR DESCRIPTION
**Pull request by romanhavrylko**
## PR Description

- Ticket: https://spryker.atlassian.net/browse/SUPESC-606

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+3. On the Storefront, go to account overview > *Newsletters*.
+4. Check that you are subscribed to the product's availability notifications.
+
+{% endinfo_block %}
+
+### 4) Set up the configuration
+
+{% info_block infoBox %}
+
+You can define stores to Yves hosts mapping. This allows generating correct URLs (e.g. unsubscription URL) for multi-store setup.
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:259]: probably store sharing should be mentioned

**Review by DmitryLymarenko — DISMISSED** [review]

**Review by KarinOf — DISMISSED** [review]

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+            new AvailabilityNotificationSubscriptionCustomerTransferExpanderPlugin(),
+         ];
+     }
+}
+```
+</details>
+
+{% info_block warningBox "Verification" %}
+
+To verify that `AvailabilityNotificationAnonymizerPlugin` is working:
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:241]:
```suggestion
To verify that `AvailabilityNotificationAnonymizerPlugin` works:
```

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 3]
```diff
@@ -0,0 +1,512 @@
...
+1. Add a new product.
+2. On the Storefront, as a company user, subscribe to its availability notifications.
+3. Check that the corresponding line is added to the `spy_availability_notification_subscription` table.</li><li>Delete this user.
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:244]: rewrite the sentence without </li><li>
```suggestion
3. Check that the corresponding line is added to the `spy_availability_notification_subscription` table.</li><li>Delete this user.
```

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 4]
```diff
@@ -0,0 +1,512 @@
...
+4. Check that the line is deleted from the `spy_availability_notification_subscription` table.
+
+To verify that `AvailabilityNotificationSubscriptionCustomerTransferExpanderPlugin` is working:
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:247]:
```suggestion
To verify that `AvailabilityNotificationSubscriptionCustomerTransferExpanderPlugin` works:
```

File: docs/pbc/all/warehouse-management-system/202304.0/install-and-upgrade/install-features/install-the-availability-notification-feature.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+---
+title: Install the Availability Notification feature
+description: The guide walks you through the process of installing the Product is Available Again feature into the project.
+last_updated: Jun 16, 2021
```
Comment (VadymSachenko) [inline docs/pbc/all/warehouse-management-system/202304.0/install-and-upgrade/install-features/install-the-availability-notification-feature.md:4]:
```suggestion
last_updated: Feb 17, 2023
```

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 6]
```diff
@@ -0,0 +1,512 @@
...
+     }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+To verify that `AvailabilityNotificationSubscriber` is working, follow these steps:
+
+1. Add a new product and make it unavailable.
+2. As a customer, subscribe to its availability notifications on Yves.
```
[the remark this answers is not in the document]
Comment (romanhavrylko) [reply _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md]: This is a part of the original IG and I guess it means Storefront.

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 7]
```diff
@@ -0,0 +1,513 @@
...
+1. Add a new product.
+2. On the Storefront, as a company user, subscribe to its availability notifications.
+3. Check that the corresponding line is added to the `spy_availability_notification_subscription` table.
+4. Delete the company user that is subscribed to the product's availability.
+5. Check that the line is deleted from the `spy_availability_notification_subscription` table.
+
+To verify that `AvailabilityNotificationSubscriptionCustomerTransferExpanderPlugin` works:
+1. Add a new product.
+2. On the Storefront, as a company user, subscribe to its availability notifications.
+3. On the Storefront, go to account overview > *Newsletters*.
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:251]:
```suggestion
3. On the Storefront, go to account overview&nbsp;<span aria-label="and then">></span> **Newsletters**.
```

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 8]
```diff
@@ -0,0 +1,513 @@
...
+         $eventSubscriberCollection->add(new AvailabilityNotificationSubscriber());
+        
+         return $eventSubscriberCollection;
+     }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+To verify that `AvailabilityNotificationSubscriber` is working, follow these steps:
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:116]:
```suggestion
To verify that `AvailabilityNotificationSubscriber` works, follow these steps:
```

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 9]
```diff
@@ -0,0 +1,513 @@
...
+            new AvailabilityNotificationSubscriptionCustomerTransferExpanderPlugin(),
+         ];
+     }
+}
+```
+</details>
+
+{% info_block warningBox "Verification" %}
+
+To verify that `AvailabilityNotificationAnonymizerPlugin` works:
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:241]:
```suggestion
Verify that `AvailabilityNotificationAnonymizerPlugin` works:
```

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 10]
```diff
@@ -0,0 +1,513 @@
...
+1. Add a new product.
+2. On the Storefront, as a company user, subscribe to its availability notifications.
+3. Check that the corresponding line is added to the `spy_availability_notification_subscription` table.
+4. Delete the company user that is subscribed to the product's availability.
+5. Check that the line is deleted from the `spy_availability_notification_subscription` table.
+
+To verify that `AvailabilityNotificationSubscriptionCustomerTransferExpanderPlugin` works:
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:248]:
```suggestion
Verify that `AvailabilityNotificationSubscriptionCustomerTransferExpanderPlugin` works:
```

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 11]
```diff
@@ -0,0 +1,513 @@
...
+| CustomerTransfer.availabilityNotificationSubscriptionSkus | column | created | src/Generated/Shared/Transfer/CustomerTransfer.php                                     |
+| MailTransfer.availabilityNotificationSubscriptionMailData | column | created | src/Generated/Shared/Transfer/MailTransfer.php                                         |
+
+{% endinfo_block %}
+
+### 3) Set up behavior
+
+Enable the following behaviors by registering the plugins.
+
+#### Listening to the availability_notification event
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:79]: Rewrite the title. It should be a task-based heading and start with a bare infinitive. For example:
```suggestion
#### Configure listening to the availability_notification event
```

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 12]
```diff
@@ -0,0 +1,513 @@
...
+Verify that `AvailabilityNotificationSubscriber` works:
+
+1. Add a new product and make it unavailable.
+2. As a customer, subscribe to its availability notifications on the Storefront.
+3. Make the product available.
+4. Check your mailbox for the email about the product's availability.
+
+{% endinfo_block %}
+
+#### Email handling
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:125]: The same. Rephrase it. For example:
```suggestion
#### Configure email handling
```

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 13]
```diff
@@ -0,0 +1,513 @@
...
+
+Verify that `AvailabilityNotificationSubscriptionMailTypePlugin`, `AvailabilityNotificationUnsubscribedMailTypePlugin`, and `AvailabilityNotificationMailTypePlugin` work:
+1. Add a new product.
+2. On the Storefront, as a customer, subscribe to its availability notifications.
+3. Switch the availability status of the product several times.
+4. Check your mailbox for emails about the product's status being switched to available and unavailable.
+
+{% endinfo_block %}
+
+#### Customer behavior
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:189]: the same. Rephrase it. For example:
```suggestion
#### Configure customer behavior
```

**Review by VadymSachenko — CHANGES_REQUESTED** [review]
See my comments please

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**